Source/ImageGlass/frmMain.cs

```diff
@@ -3394,7 +3394,8 @@ public partial class frmMain: Form {
             }
 
             if (imgIndex == Local.CurrentIndex) {
-                NextPic(0, true, true);
+                // Issue #778 : when the image is changed, do NOT keep the zoom level; it may be different for the new image
+                NextPic(0, false, true);
```

Comment:
We need to keep the zoom level because some people want to see the real time update of the region being drawn in other app (e.g. Photoshop) (requested by #168).

Reply:
No "solution" necessary. I submitted the bug for what I thought was incorrect behavior; if it is deliberate behavior for another issue, then my issue should be closed as "won't fix".